### PR TITLE
Flexible entrypoint

### PR DESCRIPTION
**Pull request by des4maisons**

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -1,8 +1,10 @@
-#!/bin/bash
+#!/bin/bash -x
+
+if [ $1 = 'prompt' ]; then
```
Comment (kevinjqiu) [inline docker-entrypoint.sh]: Just a style thing:

```
if [ "$1" = 'prompt']; then
    ...
```

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+  exec /bin/sh
+fi
 
 liquibase --driver=com.mysql.jdbc.Driver \
      --changeLogFile=/changelogs/migrations.xml \
-     --url=jdbc:mysql://${MYSQL_PORT_3306_TCP_ADDR}:${MYSQL_PORT_3306_TCP_PORT}/asm \
-     --username=root \
-     --password=${MYSQL_ENV_MYSQL_ROOT_PASSWORD} \
+     $@ \
```
Comment (kevinjqiu) [inline docker-entrypoint.sh]: `"$@"`

File: docker-entrypoint.sh (hunk 1 above)
Comment (markstory) [inline docker-entrypoint.sh]: Fear the BASH.

File: README.md [hunk 3]
```diff
@@ -9,5 +9,5 @@ Runs liquibase 2.0.5 with mysql connector
 Link this container to your mysql container, start it with your changelogs volume shared.
 
 ```
-sudo docker run -it --link db:mysql --rm -v /local/path/migrations:/changelogs freshbooks/liquibase:2.0.5
+sudo docker run -it --link db:mysql --rm -v /local/path/migrations:/changelogs freshbooks/liquibase_mysql:2.0.5
```
Comment (hutchic) [inline README.md]: whoops good catch :+1:

**Comment by des4maisons** [comment]
IRL talking with @ddub, "linking to a mysql container" should "just work" without needing to specify any options. Other use cases can be supported to, such as passing in additional command line options.

**Comment by ddub** [comment]
Awesome!

**Pull request merged**